lib/lolcommits/capturer.rb

```diff
@@ -14,3 +14,11 @@ def initialize(attributes = {})
     end
   end
 end
+
```

Comment:
Any better (cross ruby version) way to require these all at once? Also, I'd normally put all the requires at the top of the file.